.github/workflows/lint.yml

```diff
@@ -38,19 +26,6 @@ jobs:
           publishToken: ${{ secrets.SEMGREP_APP_TOKEN }}
           publishDeployment: ${{ secrets.SEMGREP_APP_DEPLOYMENT_ID }}
 
-  semgrep-run-local-rules:
```

Comment:
This is now handled with the "Semgrep Local" pre-commit hook.

Reply:
nm, I read to the end :)